l1-contracts/contracts/dev-contracts/WETH9.sol

```diff
@@ -30,7 +30,17 @@ contract WETH9 {
     function withdraw(uint256 wad) public {
         require(balanceOf[msg.sender] >= wad, "weth9, 1");
         balanceOf[msg.sender] -= wad;
-        payable(msg.sender).transfer(wad);
+        // kl todo FIXME: this is a hack so that zkfoundry works, but we are deploying WETH9 on L2 as well.
```

Comment:
I also want to mark this. zk foundry could not compile the other version

Reply:
Understandable. I agree with the change here, let's remove the `kl todo FIXME` part and just treat it as an explicit deviation